xml/ay_bigfile.xml

```diff
@@ -1093,11 +1093,11 @@ exit 0
      <listitem>
       <para>
        This option allows to enable (set to <literal>true</literal>) or
-       disable (set to <literal>false</literal> the self-update. Setting
-       this value is optional. The default is <literal>true</literal>.
+       disable (set to <literal>false</literal>) &yast; from self-updating.
```

Comment:
I'm not a native speaker, but I wonder if "enable/disable from self-updating" is good English. Shouldn't it be "prevent from"? 

What's so bad about the current wording? We could elaborate it with "YaST self-update" or "installer self-update" instead of just "self-update", but unless there is a reason to change it, I would stick to the current wording with 'enable' and 'disable'.

Reply:
OK... how about 
"enable (set to true) or disable (set to false) the YaST self-update?"
I though just saying "enable or disable self-update" was ambiguous.

Reply:
I don't really think it's ambiguous, but adding 'YaST' certainly won't hurt.